docs/modules/ROOT/pages/documentation/ecosystem/docker-images/server-image-overview.adoc

```diff
@@ -131,8 +131,8 @@ The following environment variables can be used to configure multiple settings o
 [width="100%",cols="29%,50%,21%",options="header",]
 |===
 |Name |Description |Default Value
-|`PREBOOT_COMMANDS`| The name of the file containing commands to be executed before the domain is started| `$CONFIG_DIR/post-boot-commands.asadmin` and `$CONFIG_DIR/pre-boot-commands.asadmin`
-|`POSTBOOT_COMMANDS` |The name of the file containing commands to be executed after the domain is started. This is the file written to in the `generate_deploy_commands.sh` script| `$CONFIG_DIR/post-boot-commands.asadmin`
+|`PREBOOT_COMMANDS`| The name of the file containing commands to be executed before the domain is started _Since v5.29.0, the file will not be modified by the Docker container_| `$CONFIG_DIR/post-boot-commands.asadmin` and `$CONFIG_DIR/pre-boot-commands.asadmin`
+|`POSTBOOT_COMMANDS` |The name of the file containing commands to be executed after the domain is started. This is the file written to in the `generate_deploy_commands.sh` script _Since v5.29.0, the file will not be modified by the Docker container_| `$CONFIG_DIR/post-boot-commands.asadmin`
```

Comment:
This isn't such an important change that "since" is necessary, I would remove it because it doesn't look well in the docs. There's also a missing period between the 2 sentences for each variable.

```suggestion
|`PREBOOT_COMMANDS`| The name of the file containing commands to be executed before the domain is started. The file will not be modified by the Docker container_| `$CONFIG_DIR/post-boot-commands.asadmin` and `$CONFIG_DIR/pre-boot-commands.asadmin`
|`POSTBOOT_COMMANDS` |The name of the file containing commands to be executed after the domain is started. This is the file written to in the `generate_deploy_commands.sh` script. The file will not be modified by the Docker container_| `$CONFIG_DIR/post-boot-commands.asadmin`
```